doc/variableexplorer.rst

```diff
@@ -2,95 +2,213 @@
+~~~~~
+
+For variables that are of type list, the window pane only displays the first ten values of the list. If you want to see them all, double click the variable in the pane to open a viewer that will display the index, type, size and value of each element of the list. You can also modify any value by double clicking it in the viewer and writing the new one.
+
+.. image:: images/variable_explorer/variable-explorer-list.png
    :align: center
-   :alt: List editor displaying a list of timedeltas, showing one being edited
+   :alt: List editor displaying a list, showing one being edited
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_array_2D_resize.png
+~~~~~~~~~~~~
+Numpy Arrays
+~~~~~~~~~~~~
+
+For Numpy Arrays, the window pane only shows some of the values of the array. You can open a viewer that shows you graphically in a "heat map" the values of the array, displaying each value in a colored box, depending on the size of the value. You can deactivate the background color by unchecking the option in the window. If the array is too large, background color will be automatically deactivated.
+
+.. image:: images/variable_explorer/variable-explorer-array-2D-resize.png
    :align: center
    :alt: Array editor with a 2D int array, displaying a "heatmap" of its values
 
 |
 
-|varexpcontextmenu| |histogram|
+If possible, you can also change the format of the values in the array, choosing the number of decimals that you want the array to display. For this, click the option format and in the window displayed you can set the desired format, using printf formatting. For more information go to "https://docs.python.org/3/library/stdtypes.html#printf-style-bytes-formatting".
 
-.. |varexpcontextmenu|
-   image:: images/variable_explorer/variable_explorer_inset_contextmenu_array.png
-   :width: 45%
-   :alt: Variable Explorer with a context menu, including plot/histogram options
 
-.. |histogram|
-   image:: images/variable_explorer/variable_explorer_histogram.png
-   :width: 50%
-   :alt: Plot window showing a histogram, generated via the previous options
+Additionally, you can adjust the size of the rows and columns of the array by moving the arrows. Clicking the option resize, will bring the chart back to its default size.
 
-|
+.. image:: images/variable_explorer/variable-explorer-array-resize.gif
+   :align: center
+   :alt: Array editor with a 2D int array, showing resizing of columns
+ 
+
+~~~~~~~~~~
+DataFrames
+~~~~~~~~~~
+
+For DataFrames, same as Numpy Arrays, you can see their values in a viewer where you can deactivate background color, change the format and resize the rows and columns of the table in which information is displayed. 
 
-.. image:: images/variable_explorer/variable_explorer_contextmenu_array.png
+.. image:: images/variable_explorer/variable-explorer-dataframe.png
    :align: center
-   :alt: Context menu for an int array, with the Show image option selected
+   :alt: Dataframe editor showing data frame "heatmap"
```

Comment:
We should also include an image for multi-indexed dataframes, such as the one shown in [our blog](https://www.spyder-ide.org/blog/spyder-variable-explorer/).

Reply:
Agreed.

Reply:
Done

Reply:
Great!